src/material-ui-make-payment.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="">
+  <meta name="author" content="">
+  <meta name="viewport" content="width=device-width">
+  <link rel="shortcut icon" href="https://payload.co/img/favicon.png">
+
+  <title>Payload MaterialUI Example</title>
+  <script src="https://unpkg.com/react@latest/umd/react.development.js" crossorigin="anonymous"></script>
```

Comment:
All of these scripts need to go away since (I'm assuming here) most of them are being imported in the new jsx file, right?